### PR TITLE
Chef badge

### PR DESCRIPTION
**Pull request by PichapopRo**
## What I have done
- After the user got at least 10 approved recipes. User will rewarded with a chef badge next to their name on the recipe page.

image

### CODE REVIEW

File: webpage/apps.py
Comment (knilios) [inline webpage/apps.py:1]: Please add docstring to the methods and classes.

File: webpage/signals.py
Comment (knilios) [inline webpage/signals.py:1]: Please remove the magic number '10'

File: webpage/templates/recipes/recipe_list.html
Comment (knilios) [inline webpage/templates/recipes/recipe_list.html:1]: WHAT DO YOU MEAN BY 'funny haha badge'??? PLEASE CHANGE IT.

File: webpage/utils.py
Comment (knilios) [inline webpage/utils.py:1]: Magic number '10'. Maybe this number should be set in the env file.

**Review by SariyaP — CHANGES_REQUESTED** [review]
Change this
image

File: webpage/views.py
Comment (knilios) [inline webpage/views.py:1]: Please take a lot at my comments in the pull request #155.

**Review by knilios — CHANGES_REQUESTED** [review]

File: webpage/templates/recipes/recipe_list.html
Comment (PichapopRo) [reply webpage/templates/recipes/recipe_list.html:1]: why not? la

Comment (knilios) [reply webpage/templates/recipes/recipe_list.html:1]: **no**

**Review by knilios — APPROVED** [review]

**Review by SariyaP — APPROVED** [review]

**Pull request merged**